libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.ts

```diff
@@ -629,7 +621,7 @@ export class SkyAutocompleteComponent implements OnDestroy, AfterViewInit {
 
     if (isEmpty) {
       // Emit selectionChange if value has been cleared.
-      if (this.inputDirective.value) {
+      if (this.inputDirective && this.inputDirective.value) {
```

Comment:
Can you replace `this.inputDirective && this.inputDirective.value` with `this.inputDirective?.value`?

Reply:
Good catch.